### PR TITLE
feat(dependencies): show installed version on dependency modal

### PR DESCRIPTION
**Pull request by jyash97**
## What kind of change does this PR introduce?
Enhancement

## What is the current behavior?
Currently, we don't show the installed version 

## What is the new behavior?
https://www.loom.com/share/89a6d6fbbbf1415c9c63f5c222916f23

## Checklist

- [x] Testing 
- [x] Ready to be merged

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/4943/commits/efca4c3a25d99e9d06fdf21650e03d94d0f2da5f is at https://pr4943.build.csb.dev/s/new.

**Review by SaraVieira — APPROVED** [review]
All looks good and I do think this is super nice to have!!

That said it does add things so I am gonna ask @DannyRuchtie for a review :)

**Comment by DannyRuchtie** [comment]
Hi, @jyash97 this is a really good suggestion and I really want to add it to the modal.
I suggest a slightly different way of showing it. 

Concern:
- My concern is the UI it's getting a bit cluttered (content dense) 

Suggestion: 
- My suggestion (see screenshot) is to place a gray marker in front of the dependency to indicate that you have it installed
- Have the select box show the selected version number.
- As soon as you change the version number change the gray marker to green (to indicate you selected a new one) 

Do you want to help us with building this? 
I can also ask @SaraVieira to build it
 

Screenshot 2020-09-29 at 12 56 00

**Comment by jyash97** [comment]
Glad to see that you both liked the suggestion :)

> Do you want to help us with building this?

I will be happy to help to build this. @SaraVieira will it work if I ping here if I have some issue or stuck with something?

**Comment by SaraVieira** [comment]
Pls do and we will figure it out :)

Thank you for your openness to the suggestions

**Comment by jyash97** [comment]
Changes as per the above design: 

https://www.loom.com/share/73616565ecfc4a10856d2c798326dc59

Let me know if this looks good.

File: packages/app/src/app/pages/Sandbox/SearchDependencies/Dependency.tsx [hunk 1]
```diff
@@ -98,10 +117,42 @@ export const Dependency = ({ dependency }: { dependency: DependencyType }) => {
   const getTagName = (tags: DependencyType['tags'], version: string) =>
     Object.keys(tags).find(key => tags[key] === version);
 
+  const onVersionSelect = e => {
+    const {
+      target: { value },
+    } = e;
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/Sandbox/SearchDependencies/Dependency.tsx:123]:
```suggestion
  const onVersionSelect = ({ target: { value } }) => {
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: packages/app/src/app/pages/Sandbox/SearchDependencies/Dependency.tsx (hunk 1 above)
Comment (jyash97) [reply packages/app/src/app/pages/Sandbox/SearchDependencies/Dependency.tsx:123]: Just a question, do we follow a particular code style for this? Like destructuring in param or destructing separately from a variable ?

**Comment by DannyRuchtie** [comment]
Hi, sorry for the late response. I will do a review first thing tomorrow morning. The video looks promising. Thank you so much!

**Review by DannyRuchtie — APPROVED** [review]

**Comment by SaraVieira** [comment]
Niiice!! Will merge this tomorrow when I will be at the computer 😁😁

**Comment by SaraVieira** [comment]
So that was 5 days ago and I am terrible

I am so sorry, merging now as soon as it builds :)

**Comment by SaraVieira** [comment]
Thank you so much again!!

**Comment by jyash97** [comment]
Thanks for the review. Happy that my suggestion landed in the app :)

**Pull request merged**